internal/clients/enx_redirect.go

```diff
@@ -40,6 +42,55 @@ func NewENXRedirectClient(base string, opts ...Option) (*ENXRedirectClient, erro
 	}, nil
 }
 
+func (c *ENXRedirectClient) SendUserReportIndex(ctx context.Context, apikey string, nonce string) error {
+	req, err := c.newRequest(ctx, http.MethodPost, "/report", nil)
+	if err != nil {
+		return err
+	}
+	req.Header.Set("X-API-Key", apikey)
+	req.Header.Set("X-Nonce", nonce)
+
+	res, err := c.httpClient.Do(req)
```

Comment:
Use DoOk since you just check statusok later anyway

Reply:
the doOK structure assumes API requests not HTML - I think we should start w/ this